xgboost-predictor/src/main/java/biz/k11i/xgboost/gbm/GBLinear.java

```diff
@@ -26,7 +26,7 @@ public void loadModel(PredictorConfiguration config, ModelReader reader, boolean
     }
 
     @Override
-    public float[] predict(FVec feat, int ntree_limit) {
+    public float[] predict(FVec feat, int ntree_limit, float base_score) {
```

Comment:
`base_score` is not applied in the logic. Or is it irrelevant for `GBLinear`?

Reply:
I am working on tests for Linear and Dart implementation, we don't cover it nor here nor in H2O-3.

Good point